modules/server/src/main/ts/bedrock/auto/DriverLoader.ts

```diff
@@ -56,24 +70,29 @@ const loadPhantomJs = () => {
 };
 
 export const loadDriver = (browserName: string): DriverAPI => {
-  const driverDep = browserModules[browserName];
-  if (driverDep === undefined) {
+  const driverDeps = browserModules[browserName] || [];
+  if (driverDeps.length === 0) {
     console.log('Not loading a local driver for browser ' + browserName);
   } else {
-    try {
-      return require(driverDep);
-    } catch (e) {
-      console.log(`No local ${driverDep} for ${browserName}. Searching system path...`);
+    const driver = findNpmPackage(driverDeps);
+    if (driver !== null) {
+      return driver;
+    } else {
+      console.log(`No local ${driverDeps[0]} for ${browserName}. Searching system path...`);
```

Comment:
[nitpick] should that be `${driverDeps.join(' / ')}` or something so that it shows we're definitely trying to locate the other drivers?

Reply:
I wanted it to show the primary driver only here, as the others are fallbacks and shouldn't be expected to be used.